contracts/src/v0.8/vrf/dev/VRFV2PlusWrapper.sol

```diff
@@ -84,6 +84,7 @@ contract VRFV2PlusWrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsume
   // s_wrapperGasOverhead reflects the gas overhead of the wrapper's fulfillRandomWords
   // function. The cost for this gas is passed to the user.
   uint32 private s_wrapperGasOverhead;
+  uint16 private s_wrapperGasOverheadPerWord;
```

Comment:
if this is only 7 gas units per word, we probably don't need it right? we can just add some extra buffer to s_wrapperGasOverhead.

Reply:
Sure, if my calculation is correct and the overhead is that small then we can just add some buffer for it. It's at most 2.5k more gas if it's 10 gas per word. I'll need to run more transactions.